statsmodels/stats/tests/test_lilliefors.py

```diff
@@ -0,0 +1,59 @@
+
+import numpy as np
+from numpy.testing import assert_almost_equal
+from scipy import stats
+
+from statsmodels.stats._lilliefors import lilliefors
+
+
+class TestLilliefors(object):
+
+    def test_normal(self):
+        np.random.seed(3975)
+        x_n = stats.norm.rvs(size=500)
+
+        # R function call:
```

Comment:
Are these instructions for how to generate the results via R?  +1.  In a perfect world we'd have a systematic way of externally validating test answers and not need them hard-coded (or at least to periodically double-check the hard-coded answers)

Reply:
(It looks like I never replied here.)
The perfect world is having a hardcoded Truth. (And that doesn't exist.)
A bit of history: Our first unit tests uses wrappers using rpy (precursor of rpy2) to run the equivalent models in R, and verify with those results. This was good for quickly getting up and running with the unit tests.
However, in our second year we switched to hardcoding all results from R or Stata. That avoids several problems, for example extra runtime because results need to be computed in several packages, possibly changes in the reference packages, in some parts R, Stata and statsmodels don't agree on which definition is used, or which small sample correction or ..., extra dependencies during test runs (rpy/rpy2 was not working on Windows for several years during the rpy2 development)